scripts/remove

```diff
@@ -6,85 +6,97 @@
-# REMOVE SERVICE FROM ADMIN PANEL
-#=================================================
-
-# Remove a service from the admin panel, added by `yunohost service add`
-if yunohost service status "$app" >/dev/null 2>&1
-then
-	ynh_script_progression --message="Removing $app service"
-	yunohost service remove "$app"
-fi
-
 #=================================================
 # STOP AND REMOVE SERVICE
 #=================================================
-ynh_script_progression --message="Stopping and removing the systemd service"
+ynh_script_progression --message="Stopping and removing the systemd service..." --weight=1
 
 # Remove the dedicated systemd config
 ynh_remove_systemd_config
 
 #=================================================
-# REMOVE THE POSTGRESQL DATABASE
+# REMOVE THE MYSQL DATABASE
 #=================================================
-ynh_script_progression --message="Removing the PostgreSQL database"
+ynh_script_progression --message="Removing the MySQL database..." --weight=2
 
 # Remove a database if it exists, along with the associated user
-ynh_psql_remove_db "$db_name" "$db_name"
+ynh_mysql_remove_db --db_user=$db_user --db_name=$db_name
 
 #=================================================
 # REMOVE DEPENDENCIES
 #=================================================
-ynh_script_progression --message="Removing dependencies"
+ynh_script_progression --message="Removing dependencies..." --weight=1
 
 # Remove metapackage and its dependencies
 ynh_remove_app_dependencies
 
+ynh_remove_nodejs
+
 #=================================================
 # REMOVE APP MAIN DIR
 #=================================================
-ynh_script_progression --message="Removing app main directory"
+ynh_script_progression --message="Removing app main directory..." --weight=6
 
 # Remove the app directory securely
-ynh_secure_remove "$final_path"
+ynh_secure_remove --file="$final_path"
 
 #=================================================
 # REMOVE NGINX CONFIGURATION
 #=================================================
-ynh_script_progression --message="Removing nginx web server configuration"
+ynh_script_progression --message="Removing Nginx web server configuration..." --weight=5
 
 # Remove the dedicated nginx config
 ynh_remove_nginx_config
 
+#=================================================
+# REMOVE LOGROTATE CONFIGURATION
+#=================================================
+ynh_script_progression --message="Removing logrotate configuration..." --weight=1
+
+# Remove the app-specific logrotate config
+ynh_remove_logrotate
+
 #=================================================
 # GENERIC FINALIZATION
+#=================================================
+# CLOSE A PORT
+#=================================================
+
+if yunohost firewall list | grep -q "\- $port$"
+then
+	ynh_script_progression --message="Closing port $port..."  --weight=1
+	ynh_exec_warn_less yunohost firewall disallow TCP $port
+fi
```

Comment:
I'm not sure this section is really meaningful considering we never open the port in the first place